code/__HELPERS/icons.dm

```diff
@@ -1055,8 +1055,57 @@ GLOBAL_LIST_EMPTY(friendly_animal_types)
 
 GLOBAL_LIST_INIT(freon_color_matrix, list("#2E5E69", "#60A2A8", "#A1AFB1", rgb(0,0,0)))
 
-/obj/proc/make_frozen_visual()
-	// Used to make the frozen item visuals for Freon.
+/atom/proc/begin_freeze()
```

Comment:
doc

Reply:
also all of these should go into their type paths relevant file like the atom ones to the atom level and the carbon ones to the carbon level

Reply:
also why are these on /atom level if theyre only used by /atom/movable